benchmarks/triton_kernels_benchmark/benchmark_driver.py

```
@@ -132,8 +132,8 @@ def format_of(ty):
             "long": "l",
             "int8_t": "b",
             "int16_t": "h",
-            "int32_t": "i",
-            "int64_t": "l",
+            "int32_t": "l",
```

`long int` at least 32 bits

---

The most significant bits are silently discarded.